%YAML:1.0
---
cameraMatrix: !!opencv-matrix
   rows: 3
   cols: 3
   dt: d
   data: [767.4914199999999, 0, 634.01435, 0, 771.15924, 329.31599, 0, 0, 1]
distCoeffs: !!opencv-matrix
   rows: 1
   cols: 5
   dt: d
   data: [-0.293072, 0.06916700000000001, 0.001797, 0.000445, 0]
rvec: !!opencv-matrix
   rows: 3
   cols: 1
   dt: d
   data: [ 0., 0., 0. ]
tvec: !!opencv-matrix
   rows: 3
   cols: 1
   dt: d
   data: [ 0., 0., 0. ]
